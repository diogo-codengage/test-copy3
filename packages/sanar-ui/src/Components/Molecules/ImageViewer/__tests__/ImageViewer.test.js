
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESImageViewer from '../'

it('renders correctly', () => {
    const component = (
        <ESImageViewer>
            <h1>ESImageViewer</h1>
        </ESImageViewer>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
