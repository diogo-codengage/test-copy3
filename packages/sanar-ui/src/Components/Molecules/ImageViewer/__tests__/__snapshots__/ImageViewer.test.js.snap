// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-image-viewer"
>
  <div
    className="es-image-viewer__small"
    onClick={[Function]}
  >
    <img
      src="https://i.pinimg.com/736x/28/65/74/28657471c865a566cf1347ae44bb388c.jpg"
    />
    <div
      className="es-image-viewer__small--hover"
    >
      <i
        className="anticon es-eva-icon es-eva-icon--medium"
      >
        <i
          data-eva="maximize-outline"
          data-eva-fill="#fff"
          data-eva-height="18px"
          data-eva-width="18px"
        />
      </i>
    </div>
  </div>
</div>
`;
